src/xenia/kernel/kernel_state.cc

```diff
@@ -394,6 +394,9 @@ object_ref<UserModule> KernelState::LoadUserModule(const char* raw_name,
     user_modules_.push_back(module);
   }
 
+  xe::LogLineFormat(xe::LogLevel::Error, 'i', "Title ID : %.8X\n",
```

Comment:
What's happening now for games that load multiple modules?

Reply:
It would log the title ID each time LoadUserModule is called, but then it would also log the module info in the dump function called right after so...

Reply:
That's not what I mean, are we going to get "Title ID" output for each module loaded?

Games can have multiple modules, for the relevance of this logging, we only care about the initial module loaded's title ID.

Reply:
"It would log the title ID each time LoadUserModule"

Reply:
Sorry, I wasn't clear. What I meant was, this isn't desired behavior.

Reply:
What do you propose then to handled this ? This seems the earliest part of the code where we can get title ID info.

Something like a LogOnce ?

Reply:
Why don't you do it somewhere here: https://github.com/xenia-project/xenia/blob/master/src/xenia/emulator.cc#L626 ?

Reply:
Is this part of the code called only once per game launch ?
The reason is I don't know the code base that well, so I put it somewhere where it worked

Reply:
Moved the logging where you suggested